update based on customer query

anwara-ms opened the pull request:
Added a section on what happens when a message is edited and retention is in place.

scanum commented:
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

JohanFreelancer9 commented on Teams/retention-policies.md:
```diff
@@ -60,6 +60,14 @@ To create a retention policy for Teams chats and channel messages, use the instr
 
+## Retention of edited messages
+
+Retention on Teams messages works on the timestamp of creation of original message. Lets take a scenario- If there is a policy of 30-day deletion on Teams chat or channel messages and a user sends a message (“Original message”) on day -1, later modifies it to “edited message”, on day 15.
```
```suggestion
Retention on Teams messages works on the timestamp of the creation of the original message. For example, in a scenario where there is a 30-day deletion policy on Teams chat or channel messages and a user sends a message (“Original message”) on day -1, then later modifies it to an “edited message” on day 15.
```

JohanFreelancer9 commented on Teams/retention-policies.md:
```diff
@@ -60,6 +60,14 @@ To create a retention policy for Teams chats and channel messages, use the instr
+
+Retention on Teams messages works on the timestamp of creation of original message. Lets take a scenario- If there is a policy of 30-day deletion on Teams chat or channel messages and a user sends a message (“Original message”) on day -1, later modifies it to “edited message”, on day 15.
+
+Here, the original message will be copied to a temporary location in the backend storage and deleted within 21 days. If there is Legal hold (via eDiscovery portal) or Retention hold in place on the user, then all versions of original, deleted and edited messages are stored as long as the hold policy is active.
```
```suggestion
In this scenario, the original message will be copied to a temporary location in the backend storage and deleted within 21 days. If there is a Legal hold (via the eDiscovery portal) or a Retention hold in place on the user, then all versions of original, deleted, and edited messages are stored as long as the hold policy is active.
```

JohanFreelancer9 commented on Teams/retention-policies.md:
```diff
@@ -60,6 +60,14 @@ To create a retention policy for Teams chats and channel messages, use the instr
+
+Here, the original message will be copied to a temporary location in the backend storage and deleted within 21 days. If there is Legal hold (via eDiscovery portal) or Retention hold in place on the user, then all versions of original, deleted and edited messages are stored as long as the hold policy is active.
+
+The “edited message” would be deleted by the retention policy on Day-30 i.e. based on the timestamp on the original message.
```
```suggestion
The edited message would be deleted by the retention policy on day 30, that is, based on the timestamp on the original message.
```

JohanFreelancer9 reviewed (changes requested):
Copy review - suggestions.

anwara-ms commented:
can we publish plz

scanum commented:
@cabailey PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

cabailey commented:
@scanum I see a merge conflict that need to be resolved

cabailey commented:
Closing this because I've updated this page to include concepts for edited messages (also added won't be deleted if hold also exists) & in the process of updating the M365 page where we go into detail about timings and what happens when messages are edited or deleted. I'm also looking to add a walkthrough example like this on that page, where it will be clearer where the 21 days comes from.

anwara-ms commented:
Hi Carol
Can you please tell me what exactly has been updated re: edited messages? The customer confusion was specifically about timelines of edited messages and I believe that is not addressed.

Please, I want the original message to be posted to Teams documentation. Roberto started a thread on this 21 day delay and he got clarification from chatservice. I'll fwd that msg as FYI to you.


  --

Anwar | PM @ Teams  IP + Compliance | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>



________________________________
From: Carol Bailey ***@***.***>
Sent: Wednesday, April 7, 2021 05:19 PM
To: MicrosoftDocs/OfficeDocs-SkypeForBusiness ***@***.***>
Cc: Anwar Ali ***@***.***>; Author ***@***.***>
Subject: Re: [MicrosoftDocs/OfficeDocs-SkypeForBusiness] update based on customer query (#6644)


Closing this because I've updated this page to include concepts for edited messages (also added won't be deleted if hold also exists) & in the process of updating the M365 page where we go into detail about timings and what happens when messages are edited or deleted. I'm also looking to add a walkthrough example like this on that page, where it will be clearer where the 21 days comes from.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F6644%23issuecomment-815357172&data=04%7C01%7CAnwar.Ali%40microsoft.com%7C041cbead551c40d6a12108d8fa23ffd1%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637534379852329448%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=l%2B0gYtqLYTZpo1qZ0A03nOpjpLNy8lFlsgvL%2FHwAaC4%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACEEVXZO2THAJRRWUOWF5OLTHTZBXANCNFSM4XNIJUZA&data=04%7C01%7CAnwar.Ali%40microsoft.com%7C041cbead551c40d6a12108d8fa23ffd1%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637534379852329448%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=3%2F8JM1%2FY3I7SRm15h82X60AB2z4GW1zBXB%2BLPqrO61A%3D&reserved=0>.

cabailey commented:
Continue conversation internally.
